src/Plugin/BlockSettings/EntityTypeBundleSelectionTrait.php

```diff
@@ -96,13 +96,15 @@ public function getEntityTypes() {
    *   List of bundles.
    */
   public function getBundles($target_type) {
-    $options = \Drupal::entityManager()
+    $options = \Drupal::entityTypeManager()
       ->getBundleInfo($target_type);
```

Comment:
I think `entityTypeManager` does not have any function getBundleInfo().
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityManager.php/function/EntityManager%3A%3AgetBundleInfo/8.9.x

Reply:
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityTypeBundleInfo.php/function/EntityTypeBundleInfo%3A%3AgetBundleInfo/8.2.x

It is moved to a different module. Please check.

Reply:
Addressed in the latest commit.